test/persistence/abstract.js

```diff
@@ -558,7 +558,7 @@ module.exports = function(create, buildOpts) {
       });
     });
 
-    it("should not store a QoS 0 subscription", function(done) {
+    it("should store a QoS 0 subscription", function(done) {
```

Comment:
can you please check/add a test that we are not enqueuing QoS 0 packets?

Reply:
Can you tell me what you mean? You already have test for QoS 0 packages

Reply:
forget my comment :)

Reply:
Eeeee... what comment???? ;)